SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -50,6 +50,7 @@
     PhysicsTablesDirectory = cms.untracked.string('PhysicsTables'),
     StorePhysicsTables = cms.untracked.bool(False),
     RestorePhysicsTables = cms.untracked.bool(False),
+    UseParametrisedEMPhysics = cms.untracked.bool(False),
```

Comment:
should this be true by default to keep previous behavior?

Reply:
You are right - corrected